docs/source/quark_mit.rst

```diff
@@ -0,0 +1,43 @@
++++++++++++++++++
+Quark MIT Program
++++++++++++++++++
+
+Quark MIT aims to create a **WIN-WIN** for both new comers and the community. 
+
+For new comers, the community helps you to build a stronger resume 
+by creating specific works that fit the job description of your **DREAM JOB**.
+
+| For the community, Quark-Engine gets new energy by the work the new comers contribute. 
+| And the most important of all, the Quark community gets to **GROW**.
+
+- **Find more details below:**
+
+.. image:: https://i.imgur.com/xXilFs8.png
```

Comment:
You should use this image since this is the doc that details everything.
![Quark MIT Program Card](https://user-images.githubusercontent.com/593954/193989701-ecb409af-9486-40b9-aabc-d2f5be6032cf.png)

The image you use if only for our twitter release.